### ISSUE TITLE
Lighting shaders

### ISSUE THREAD

**Issue by christiancosgrove** [BoolClub/ProjectCrusade#7]
Implement shaders for per-pixel lighting.

**Comment by christiancosgrove** [comment]
https://github.com/mono/MonoGame/wiki/Effects-And-Shaders

**Comment by christiancosgrove** [comment]
http://stackoverflow.com/questions/23733470/monogame-and-fx-files

**Comment by AnonymousNix** [comment]
If we could figure out how to properly implement the "experimental graphics", We may be able to do this.

**Comment by christiancosgrove** [comment]
Yeah, that's not really the issue. You mentioned a problem importing shaders--that's the main issue. I would probably be able to write necessary shaders to do this.

**Comment by christiancosgrove** [comment]
It's mostly an issue of figuring out how to import shaders into MonoGame, which in theory behaves the same as XNA, but in practice requires a bit of extra work (frustratingly). One example is importing SpriteFonts, which is usually a very simple process.
